connectivity/drivers/lora/COMPONENT_SX1276/SX1276_LoRaRadio.cpp

```diff
@@ -1263,13 +1266,27 @@ void SX1276_LoRaRadio::read_fifo(uint8_t *buffer, uint8_t size)
 void SX1276_LoRaRadio::set_operation_mode(uint8_t mode)
 {
     if (mode == RF_OPMODE_SLEEP) {
+        write_to_register(REG_OPMODE, (read_register(REG_OPMODE) & RF_OPMODE_MASK) | mode);
         set_low_power_mode();
+        /* FIXME taken from set_modem, it reduces the power consumption
+         *from 300µA to 1.5µA */
+        write_to_register(REG_DIOMAPPING1, 0x00); // sets DIO0-DI03 in default mode
+        write_to_register(REG_DIOMAPPING2, 0x30); // bits 4-5 are turned on i.e.,
+        if (_rf_ctrls.tcxo != NC) {
+            ThisThread::sleep_for(1ms);
+            _tcxo = 0;
+        }
     } else {
+        if (_rf_ctrls.tcxo != NC) {
+            _tcxo = 1;
+            // TCXO startup time
+            ThisThread::sleep_for(5ms);
```

Comment:
From the tests I did on a custom Murata-based board, if this period is greater or equal to 9ms, the device is unable to receive a join accept.

Reply:
Thanks @mickael868, this mean that 5ms looks like correct value (no magic, I took this 5ms from code of other stack).
Pay attention to SPI, SPI communication with sx1272 won't work if TCXO is not enabled. Worth mentioning this one